controllers/ccruntime_controller.go

```diff
@@ -847,3 +878,33 @@ func (r *CcRuntimeReconciler) makeHookDaemonset(operation DaemonOperation) *apps
 		},
 	}
 }
+
+func (r *CcRuntimeReconciler) removeNodeLabels(nodesList *corev1.NodeList) (ctrl.Result, error) {
+	nodesClient, err := r.getNodeClient()
+	if err != nil {
+		r.Log.Info("Couldn't get nodes client")
+		return ctrl.Result{}, err
+	}
+
+	for _, node := range nodesList.Items {
+		nodeLabels := node.GetLabels()
+		for k, v := range nodeLabels {
+			if k == PostUninstallDoneLabel[0] && v == PostUninstallDoneLabel[1] ||
+				k == PreInstallDoneLabel[0] && v == PreInstallDoneLabel[1] ||
+				k == "katacontainers.io/kata-runtime" && v == "cleanup" {
```

Comment:
How about defining the above runtime specific label also in common.go ? 
Keeping all the required labels in one place for easier reference

I want to suggest a style change with an example. It will make the code easier to read when adding new labels for different runtimes. 
```
if val, ok := nodeLabels[PreInstallDoneLabel[0]]; ok && val == PreInstallDoneLabel[1] {
          delete(nodeLabels, PreInstallDoneLabel[0])
 }

if val, ok := nodeLabels[PostUninstallDoneLabel[0]]; ok && val == PostUninstallDoneLabel[1] {
           delete(nodeLabels, PostUninstallDoneLabel[0])
}

if val, ok := nodeLabels[KataRuntimeLabel[0]]; ok && val == KataRuntimeLabel[1] {
          delete(nodeLabels, KataRuntimeLabel[0])
}
```

Reply:
good suggestion, I'll adapt it accordingly!